src/components/TaskStatusIcon/index.tsx

```diff
@@ -0,0 +1,46 @@
+import { uiColors } from "@leafygreen-ui/palette";
+import Icon from "components/Icon";
+import { TaskStatus } from "types/task";
+import { errorReporting } from "utils";
+
+const { reportError } = errorReporting;
+const { green, red, yellow, gray } = uiColors;
+
+interface TaskStatusIconProps {
+  status: TaskStatus;
+}
+
+export const TaskStatusIcon: React.FC<TaskStatusIconProps> = ({ status }) => {
+  switch (status) {
+    case TaskStatus.Succeeded:
+      return <Icon glyph="Checkmark" fill={green.base} size={30} />;
+    case TaskStatus.Failed:
+      return <Icon glyph="X" fill={red.base} size={30} />;
+    case TaskStatus.KnownIssue:
+      return <Icon glyph="KnownFailureIcon" fill={red.base} size={30} />;
+    case TaskStatus.Dispatched:
+    case TaskStatus.Started:
+      return <Icon glyph="Refresh" fill={yellow.dark2} size={30} />;
+    case TaskStatus.SetupFailed:
+      return <Icon glyph="SetupFailure" size={30} />;
```

Comment:
[nit] It may be preferable to have consistent usage of the `fill` prop among all icons (e.g. have Spruce custom icons accept `fill` in the same manner as LG), but definitely not a necessary change.

Reply:
nice catch. this will be a useful change to follow updates in @leafygreen-ui/palette